setup.py

```diff
@@ -25,6 +25,15 @@ def read(*parts):
     },
     test_suite="runtests.runtests",
     tests_require=[
+        "django-test-plus>=1.0.11",
+        "pinax-theme-bootstrap>=7.4.0",
+        "django-appconf>=1.0.1",
+        "mock>=1.3.0",
+        "django-user-accounts>=1.3.1",
+    ],
+    install_requires=[
+        "django-appconf>=1.0.1",
+        "django-user-accounts>=1.3.1",
```

Comment:
It's to my understanding that `install_requires` is always installed because to run tests it does an install first. Then `tests_require` are installed to ensure running tests will work correctly. So, if a dependency is needed only for tests then it only needs to appear in `tests_require`.